### ISSUE TITLE
Remove all non-OL related CSS, replace with MUI's CssBaseline

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#203]
We should get rid of most stuff in index.css. We can not remove it entirely as it is needed to style some OpenLayers components. But anything else should be done using the MUI approach.

MUI provides a [CssBaseline component](https://material-ui.com/style/css-baseline/) that does some normalization. Enabling it now and removing the CSS file clearly shows where we've missed to use MUI components:
![image](https://user-images.githubusercontent.com/110222/56805463-71944100-6829-11e9-880e-b594bd2552f9.png)

**Comment by jacobwod** [comment]
Do the following to prepare your local setup for testing:
1. `import { CssBaseline } from "@material-ui/core";`
1. Add `<CssBaseLine />` as the first thing that gets rendered in App.js's `render()`. (Currently that's just prior to the `<Alert>` component.)
1. Replace existing `index.css` with something like this:
```
/* Used only for displaying error if nothing loaded (and we don't have any MUI yet) */
.start-error {
  color: red;
  background: rgb(255, 200, 200);
  border: 1px solid red;
  border-radius: 5px;
  padding: 15px;
  margin: 0 auto;
  width: 50%;
  text-align: center;
  margin-top: 5%;
}

/* OpenLayers styling */
.ol-zoom,
.ol-full-screen,
.ol-rotate {
  top: 8px;
  left: inherit;
  right: 0px;
}

.ol-full-screen {
  top: 98px;
}

.ol-rotate {
  top: 175px;
}

.ol-control {
  border-radius: 0;
  padding: 0;
  background: 0;
}

.ol-control:hover,
.ol-control:focus {
  background: inherit;
}

.ol-control button {
  background: inherit;
  color: inherit;
  font-size: inherit;
  font-weight: inherit;
  cursor: inherit;
  margin: inherit;
  box-shadow: inherit;
}

.ol-control button:hover,
.ol-control button:focus {
  background: inherit;
}

.ol-scale-line {
  left: inherit;
  right: 8px;
}

.ol-overlaycontainer-stopevent {
  z-index: inherit !important;
}

.ol-overlay-container {
  z-index: 2000 !important;
}

@media only screen and (max-width: 600px) {
  .tool-panel {
    position: fixed;
    top: 0px;
    left: 0px;
    right: 0;
    left: 0;
    bottom: 0;
    background: white;
    padding: 10px;
    overflow-y: auto;
    border: 1px solid #ccc;
    border-radius: 0px;
    width: inherit;
  }
  .ol-zoom {
    bottom: 20px;
    top: 20px;
  }
}
```

**Comment by jacobwod** [comment]
…and it's fixed as of 5c270f92bfaa146ea92d3fe4dfee17f22fc6fe48.
